plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/machine/MachineStatusHandler.java

```diff
@@ -119,6 +120,12 @@ private void handleMachineRunning(final String machineId, final WorkspaceRuntime
         if (machine == null) {
             return;
         }
-        eventBus.fireEvent(new MachineStateEvent(machine, RUNNING));
+
+        try {
+            eventBus.fireEvent(new MachineStateEvent(machine, RUNNING));
```

Comment:
I don't know how much handlers handle the event and few developers handle exceptions in their code ( null pointer exception for example ). So here I have to guarantee the event will be fired and following code will be executed.